models/post.py

```
@@ -0,0 +1,39 @@
+
+class Post(BaseModel):
+    title: str
+    content: str
+    media : Optional[HttpUrl] = None
+    created_at: datetime
+
+def post_form_reader(
+        title : Annotated[str, Form(...)],
+        content : Annotated[str, Form(...)],
+        media : Optional[Annotated[str, Form(...)]] = None,
+        created_at: datetime = datetime.now()
```

Ruff (0.14.11)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)